src/main/java/com/KacFlor/ShopSpring/controller/CardDataController.java

```diff
@@ -13,17 +16,17 @@
 @RequestMapping(
         path = {"carddata"}
```

Comment:
Change "carddata" to "card-data". See https://restfulapi.net/resource-naming/

Reply:
done